pkg/deploy/api/validation/validation_test.go

```diff
@@ -365,6 +365,40 @@ func TestValidateDeploymentConfigMissingFields(t *testing.T) {
 			fielderrors.ValidationErrorTypeRequired,
 			"template.strategy.rollingParams.pre.failurePolicy",
 		},
+		"small template.strategy.rollingParams.updatePercent": {
```

Comment:
Shouldn't we have tests that pass somewhere? If you add those, test using the limits.